mysql-test/t/subselect4.test

```diff
@@ -2045,3 +2045,26 @@ SELECT 2 IN (SELECT 2 from DUAL WHERE 1 != 1);
 
 SET optimizer_switch= @@global.optimizer_switch;
 set @@tmp_table_size= @@global.tmp_table_size;
+
+--echo #
```

Comment:
Hello,
I only meant to add the single change which was to add one word to an error message.  I don't know where the other commit is coming from, nor do I know how to remove it.  If you could please help me with that, it would be greatly appreciated, as it is quite confusing.  Next, I do have a test which generates the error, and tested it locally to verify it works as expected:


MariaDB [test]> select version();
--
+----------------+
\| version()      \|
+----------------+
\| 5.5.64-MariaDB \|
+----------------+
1 row in set (0.00 sec)

CREATE TABLE test.table1 (
     a INT NOT NULL,
     b VARCHAR(32),
     c INT AS (a mod 10) VIRTUAL,
     d VARCHAR(5) AS (left(b,5)) PERSISTENT);

INSERT INTO test.table1 VALUES (2, 'more text',5,default);

ERROR 1906 (HY000): The value specified for computed column 'c' in table 'table1' is ignored

Reply:
Or can I somehow wipe out this whole change/commit and start fresh somehow?

Reply:
@chriscalender :

To clean up this PR do the following changing the `remote_name` to whatever remote you have configured for your repository:
<pre>
git checkout -b 5.5-MDEV-17449_typo origin/5.5
git cherry-pick 59160a2
git push -f remote_name 5.5-MDEV-17449_typo:MDEV-17449
</pre>